### ISSUE TITLE
Android: Blank screen shows when clicked on the hardware back button after the Camera opens up

### ISSUE THREAD

**Issue by ahsanhabib23** [AppliedRecognition/Ver-ID-Credentials-Cordova-Plugin#26]
Steps to produce: 
1. Open the AR.
2. Click on **SCAN FRONT OF CARD** button
3. The Camera opens up. (May need to allow camera permission)
4. The click the Android **hardware back button**
**Observe:** A blank screen is showing
5. If we click the back button again, the first screen appears. 

So the issue is the **blank screen** in between

[https://www.youtube.com/watch?v=l1SGt2UqktA](https://www.youtube.com/watch?v=l1SGt2UqktA)

[![AR](https://img.youtube.com/vi/l1SGt2UqktA/0.jpg)](https://www.youtube.com/watch?v=l1SGt2UqktA)

```
Ionic:

   ionic (Ionic CLI)  : 4.8.0 (/usr/local/lib/node_modules/ionic)
   Ionic Framework    : ionic-angular 3.9.5
   @ionic/app-scripts : 3.2.2

Cordova:

   cordova (Cordova CLI) : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms     : android 7.1.4
   Cordova Plugins       : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 4.0.1, (and 6 other plugins)

System:

   Android SDK Tools : 26.1.1 (/Users/bs228/Library/Android/sdk)
   ios-deploy        : 1.9.4
   NodeJS            : v10.15.1 (/usr/local/bin/node)
   npm               : 6.4.1
   OS                : macOS High Sierra
   Xcode             : Xcode 10.1 Build version 10B61
```

**Comment by jakubdolejs** [comment]
Thank you @ahsanhabib23. We're looking into it.

**Comment by jakubdolejs** [comment]
Fixed in 05653e387898c28a69998c8899d62e4c1501630e
